win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1454,7 +1454,22 @@ CURSORICON_LoadImageW(
             RtlInitUnicodeString(&ustrRsrc, lpszName);
     }
 
-    if(hinst)
+    if(LDR_IS_RESOURCE(hinst))
+    {
+        /* We don't have a real module for GetModuleFileName, construct a fake name instead.
+        ** GetIconInfoEx reveals the name used by Windows.
+        */
+        LPCWSTR fakeNameFmt = sizeof(void*) > 4 ? L"\001%016IX" : L"\001%08IX";
```

Comment:
`\0 ....` <-- is it understood as a NULL character? or is it understanding it as "001" octal number?
because if it's a NULL character as the first one, I'm not sure how the wsprintfW call below could work.